### DISCUSSION TITLE
Welcome to ADYPSOE AI&DS Discussions !!!

### DISCUSSION THREAD

**Discussion by gaikwadshantanu12** [gaikwadshantanu12/adypsoe_aids#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community (especially those students under the category of "Department of Artificial Intelligence and Data Science" affiliated to "Savitribai Phule Pune University"). We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Discussions on subject wise practicals right from Second Year to Final Year.
   * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gunjanrane12** [comment]
Please push the mini project code too ..
